.happy/terraform/modules/batch/main.tf

```diff
@@ -10,7 +10,7 @@ resource aws_batch_job_definition batch_job_def {
 {
   "jobRoleArn": "${var.batch_role_arn}",
   "image": "${var.image}",
-  "memory": 28000,
+  "memory": 230000,
```

Comment:
1) Can/should we make this configurable per environment?
2) Let's update the vcpu setting to match the old config as well.

Reply:
vcpu setting is same as old config. i'll make this configurable per environment.

Reply:
Random question -- is this field not deprecated? https://docs.aws.amazon.com/batch/latest/APIReference/API_ContainerProperties.html#Batch-Type-ContainerProperties-memory

Reply:
@maniarathi for ec2 resources it is still supported. not for fargate though.